metronome/hotstuff/service/src/io/iohk/metronome/hotstuff/service/CryptoUtils.scala

```diff
@@ -0,0 +1,49 @@
+package io.iohk.metronome.hotstuff.service
+
+import org.bouncycastle.asn1.sec.SECNamedCurves
+import org.bouncycastle.asn1.x9.X9ECParameters
+import org.bouncycastle.crypto.AsymmetricCipherKeyPair
+import org.bouncycastle.crypto.generators.ECKeyPairGenerator
+import org.bouncycastle.crypto.params.{
+  ECDomainParameters,
+  ECKeyGenerationParameters,
+  ECPublicKeyParameters
+}
+import scodec.bits.BitVector
+
+import java.security.SecureRandom
+
+// TODO: Remove after publishing crypto stuff from mantis
```

Comment:
I managed to publish the [snapshot](https://oss.sonatype.org/#nexus-search;quick~mantis-crypto). A release is going to take a while I guess, but we can start using these.